_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Improving matters is as easy as heading over to [Weblate](https://hosted.weblate.org/engage/newpipe/) to fix even the source strings.
 
 ## Fixes
 
-If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+Next to the now working offline search input, the actual text of the suggestions look better now. [@TobiGr](https://github.com/TobiGr) changed the colours to the NewPipe palette.
 
-An empty error report in certain kinds of crashes is not empty anymore, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code! He also fixed a crash when NewPipe encounters an empty comment. Now it will be displayed just as empty as it is.
+A spooky empty error report in certain kinds of crashes will no longer be empty, courtesy of [@Stypox](https://github.com/Stypox) taking care of properly calling the code. Yay, code! He also fixed a crash when NewPipe encounters an empty comment. Now it will be displayed just as empty as it is. Ding an sich.
 
-Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab (and potentially other locations), whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
+Instead of causing a crash, thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab (and potentially other locations). Thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, it goes by that name, and not `Tamaziɣt Taqbaylit`.
 
-The license pop-up doesn't disappear when you change orientation now. This was introduced by [@nmurali94](https://github.com/nmurali94). And [@budde92](https://github.com/budde25) ensured that you can now delete a file you just downloaded and then try to re-download it without the app crashing. We are not sure why you'd want to do this, but now you can do so without any worries.
+The license pop-up now lacks the ability to just disappear when you change orientation. This was fixed by [@nmurali94](https://github.com/nmurali94).
+[@budde92](https://github.com/budde25) clicked enough keyboard to where you can now delete a file you just downloaded _and_ re-download it _without_ the app crashing. In what is a conundrum, we are not sure why you'd want to before now, but do.
 
-[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Yay for less redundancy. When auto-queue is enabled, Newpipe wants to add a stream to the queue the moment you open a video. But if there are no video suggestions on that page, it causes a crash. This has been fixed by [@wb9688](https://github.com/wb9688).
```

Comment:
OK, meant to know what that means? Pretty grim, what is exciting and why? Give it something.

Reply:
> Yay for less redundancy. 

We've used 'yay' before as well. The change itself is boring, and the reader knows it, so we inject some humour this way.

Reply:
Yay